tonic/pycompat.py

```diff
@@ -0,0 +1,45 @@
+import sys
+
+PY3 = sys.version_info[0] >= 3
+
+if PY3:  # pragma: no cover
+    basestring = str
+    unicode_type = str
+    bytes_type = bytes
+
+    def iteritems(d):
+        return iter(d.items())
+
+    def itervalues(d):
+        return iter(d.values())
+
+    pyrange = range
+    pyzip = zip
```

Comment:
Should this be moved out-of (or to before) the 'if PY3' because if Python2 is being used (not PY3) then 'pyzip' won't be defined.